doc/classes/RenderingServer.xml

```diff
@@ -2046,6 +2046,13 @@
 				Sets whether an instance is drawn or not. Equivalent to [member Node3D.visible].
 			</description>
 		</method>
+		<method name="instance_teleport">
+			<return type="void" />
+			<param index="0" name="instance" type="RID" />
+			<description>
+				Resets motion vectors and other interpolated values. Use this [i]after[/i] teleporting a mesh from a position to another to avoid ghosting artifacts.
```

Comment:
Not a biggie, but it's slightly more expected native english to say "from one position to another" rather than "from a position to another" (both are semantically correct thought).